samples/drivers/counter/alarm/src/main.c

```diff
@@ -23,6 +23,7 @@ static void test_counter_interrupt_fn(struct device *counter_dev,
 	u64_t now_usec = counter_ticks_to_us(counter_dev, now_ticks);
 	int now_sec = (int)(now_usec / USEC_PER_SEC);
 	struct counter_alarm_cfg *config = user_data;
+	int err = 0;
```

Comment:
Why err is initialized here?

Reply:
Blindly copied code from the main function below, but not needed indeed.